jsk_naoqi_robot/README.md

```diff
@@ -13,12 +13,12 @@ rosdep install -y -r --from-paths src --ignore-src
 catkin build
 source devel/setup.bash
 ```
-% Make sure that you have already installed the ``Python NAOqi SDK`` in your computer. If not, you can download it from [here](https://community.aldebaran.com/en/resources/software). After downloading the file, unzip and rename it to ``pynaoqi``, then put it under your home folder.
+% Make sure that you have already installed the ``Python NAOqi SDK`` in your computer. If not, you can download it from [here](https://community.aldebaran.com/en/resources/software). After downloading the file, unzip and rename it to ``pynaoqi``, then put it under your home folder. And then, please add python path to your ``.bashrc`` like ``export PYTHONPATH=$HOME/pynaoqi/<your naoqi sdk version>:$PYTHONPATH``. 
  
 You need to set NAO_IP and ROS_IP environment variable to launch `jsk_pepper_startup.launch`
 ```
 source ~/catkin_ws/devel/setup.bash
-export NAO_IP="olive.jsk.imi.i.u-tokyo.ac.jp"
+export NAO_IP="olive.jsk.imi.i.u-tokyo.ac.jp" % OR IP address like "133.11.216.xxx"
 export ROS_IP="133.11.216.xxx" % OR run rossetip command to set ROS_IP
```

Comment:
since ROS_IP and NAO_IP may differ in many cases, better to set 133.11.216.xxx for ROS_IP and 133.11.216.yyy for NAO_IP, is it correct?